terranova/commands/binds.py

```diff
@@ -277,9 +277,27 @@ def docs(docs_dir: Path) -> None:
     default=False,
     is_flag=True,
 )
+@click.option(
+    "--detailed-exitcode",
+    help="""
+    \b
+    Return detailed exit codes when the command exits.
+    This will change the meaning of exit codes to:
+    0 - Succeeded, diff is empty (no changes)
+    1 - Errored
+    2 - Succeeded, there is a diff
+    """,
+    is_flag=True,
```

Comment:
Do we plan to enable this by default? If so, will it be a breaking change? I wonder whether this should be noted somehow so consumers know this change of behaviour, maybe a bump of minor version?

Reply:
`is_flag=True` does not enable it by default. It tells click that it's a boolean flag where you can omit the value

Reply:
Thanks for the explanation! Then no breaking change, 👍